grpc-example/grpc-example-java/build.sbt

```diff
@@ -42,10 +39,6 @@ lazy val `hello-impl` = (project in file("hello-impl"))
     ),
   akkaGrpcExtraGenerators in Compile += PlayJavaServerCodeGenerator,
 
-  // WORKAROUND: Lagom still can't register a service under the gRPC name so we hard-code
-  // the port and the use the value to add the entry on the Service Registry
-  lagomServiceHttpsPort := `hello-impl-HTTPS-port`,
```

Comment:
I think this won't work. We must force the `lagomServiceHttpPort` (or whatever the setting name is for HTTP ports) for the port of the backend service to a well-known value.

Removing this but keeping `lagomUnmanagedServices in ThisBuild := Map("helloworld.GreeterService" -> s"http://127.0.0.1:11000")` means the resolution won't work.

Reply:
works on my machine

Reply:
Sorry, you are right. It worked on my machine because I was testing the maven variant. 🤦

Reply:
"Works on my machine when I used in a completely different way with different tools".